Utilities/BEType.h

```diff
@@ -1,4 +1,4 @@
-#pragma once // No BOM and only basic ASCII in this header, or a neko will die
+﻿#pragma once // No BOM and only basic ASCII in this header, or a neko will die
```

Comment:
You added BOM, neko will die.

Reply:
@Nekotekina I used vs, need to reconfigure it?

Reply:
https://github.com/RPCS3/rpcs3/pull/9346#discussion_r534337342
answer the question, @Nekotekina

Reply:
You better do to stop it adding BOM everywhere.
But situation is complicated, we need BOM in SOME files.
I added comments where only BOM should not be added.